pkg/observability/k8s-events.go

```diff
@@ -113,7 +113,7 @@ func InitK8sEventRecorder(enabled bool, nodeName string, sqsMode bool, nodeMetad
 	}
 
 	broadcaster := record.NewBroadcaster()
-	broadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: clientSet.CoreV1().Events("default")})
+	broadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: clientSet.CoreV1().Events("")})
```

Comment:
@snay2 asked https://github.com/aws/aws-node-termination-handler/pull/631/files#r868349413

> Can you provide some context on this change? What is the difference in behavior between `"default"` and `""`?

Haven't checked the code but I'm fairly sure they're equivalent. In fact I that `"default"` comes from my previous PR https://github.com/aws/aws-node-termination-handler/pull/411/files#diff-204929b469b9bbc88ae1116388da4e13f00f385c77c1db140d842371ab7e79f6R116 and then @dthorsen changed it to `""`. Maybe he can answer your question.